configuring.md

```diff
@@ -127,11 +129,15 @@ android {
       // needs account information.
       systemProperty 'robolectric.dependency.repo.username', 'username'
       systemProperty 'robolectric.dependency.repo.password', 'password'
+      systemProperty 'robolectric.dependency.proxy.host', project.findProperty("systemProp.https.proxyHost") ?: System.getenv("ROBOLECTRIC_PROXY_HOST")
+      systemProperty 'robolectric.dependency.proxy.port', project.findProperty("systemProp.https.proxyPort") ?: System.getenv("ROBOLECTRIC_PROXY_PORT")
     }
   }
 }
 ```
 
+
```

Comment:
IMO, we don't need space here.

Reply:
Removed 😄! I was gonna add a section but finally chose otherwise.